AdventOfCode2021/day4_1/day4_1_2.py

```diff
@@ -0,0 +1,133 @@
+#!python3
+
+from collections import namedtuple
+from typing import List, Dict, NamedTuple
+
+
+class MatrixPoint(NamedTuple):
+    matrix_num: int
+    row_num: int
+    col_num: int
+
+
+def file_read(file_name) -> List[str]:
+    file_data_list: List[str] = []
+    with open(file_name) as f:
+        for line in f:
+            file_data_list.append(line.strip())
+    return file_data_list
+
+
+def create_bingo_card_dictionary(dictionary: Dict, key: str, value: MatrixPoint) -> Dict:
+    dict_value: List[MatrixPoint] = []
+    if (key in dictionary):
+        dict_value = list(dictionary[key])
+        dict_value.append(value)
+        dictionary[key] = dict_value
+    else:
+        dict_value.append(value)
+        dictionary[key] = dict_value
+    return dictionary
```

Comment:
There is a much simpler way to express this logic:
```py
def create_bingo_card_dictionary(dictionary: Dict, key: str, value: MatrixPoint):
    if key not in dictionary:
        dictionary[key] = []
    dictionary[key].append(value)
```
we can discuss why it's the same thing in person.